app/Jobs/DeleteQueryserviceNamespaceJob.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Jobs;
+
+use App\QueryserviceNamespace;
+use App\Http\Curl\CurlRequest;
+use App\Http\Curl\HttpRequest;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+
+class DeleteQueryserviceNamespaceJob extends Job implements ShouldBeUnique
+{
+    private $wikiId;
+    private $request;
+
+    /**
+     * @return void
+     */
+    public function __construct( $wikiId, HttpRequest $request = null)
```

Comment:
HttpRequest can't be injected here (as discovered today)

Reply:
done